README.md

```diff
@@ -1,4 +1,4 @@
-DLMFLDSM,L,LS# testo nuovo file
+fCIOOOOODLMFLDSM,L,LS# testo nuovo file
```

Comment:
Meno manie di megalomania e grandezza, grazie.
P.S il passaggio in auto lo voglio che piove

Reply:
carissimo paul fottuto jicmon, al momento sto valutando se perseverare in questa giornata didattica o alzare bandiera bianca, sono rattristito che non sia soddisfatto del mio operato da megalomane, appena ha un minuto se vuole glielo spiego dato che lei non sa fare un cazzo.

p.s. il passaggio puo ficcarselo nel culo

Reply:
p.p.s. ti voglio bene anch'io